pkg/utils/has/controller.go

```diff
@@ -430,12 +431,14 @@ func (h *SuiteController) WaitForComponentPipelineToBeFinished(c *common.SuiteCo
 		for _, condition := range pipelineRun.Status.Conditions {
 			GinkgoWriter.Printf("PipelineRun %s reason: %s\n", pipelineRun.Name, condition.Reason)
 
-			if condition.Reason == "Failed" {
-				return false, fmt.Errorf(tekton.GetFailedPipelineRunLogs(c, pipelineRun))
+			if !pipelineRun.IsDone() {
+				return false, nil
```

Comment:
Why are we not getting logs in this case? (I just might not fully understand the underlying bug)

Reply:
ack. thanks ;-)